_data/cloud.yml

```diff
@@ -62,15 +64,17 @@ websites:
     tfa:
       - sms
       - phone
-      - software
+      - totp
+      - proprietary
+      - u2f
       - hardware
```

Comment:
Is the need to keep `hardware` given as you added `u2f` - because you remove `software`?